README.md

```diff
@@ -118,7 +118,7 @@ Allows to destroy the session in the store. If you do not pass a callback, a Pro
 
 Updates the `expires` property of the session's cookie.
 
-#### Session#regenerate(callback)
+#### Session#regenerate(callback, [ignoreFields])
```

Comment:
`callback` should place at the last field.
We can check if the first field is `Array` or `Function` to prevent breaking changes.